cms_form_example/README.rst

```diff
@@ -12,8 +12,12 @@ Basic content type form example.
 Usage
 =====
 
-TODO
+Go on the website app of Odoo. Then you can reach the following pages:
```

Comment:
pls appy this changes:
"""
Go on the website app of Odoo. Then you can reach the following routes:

* `/partners`: shows you a nice form to search partners
* `/partners/add`: shows you a nice form to add a new partner
* `/partners/$ID/edit`: shows you a nice form to edit partner with matching $ID
"""